subsys/can/isotp/isotp.c

```diff
@@ -0,0 +1,1247 @@
+/*
+ * Copyright (c) 2019 Alexander Wachter
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "isotp_internal.h"
+#include <net/buf.h>
+#include <kernel.h>
+#include <init.h>
+#include <misc/util.h>
+
+#define LOG_LEVEL CONFIG_CAN_ISOTP_LOG_LEVEL
+#include <logging/log.h>
+LOG_MODULE_REGISTER(isotp);
```

Comment:
Unless you use the LOG_LEVEL for something in the code, an alternative way of setting the log level is just to have `LOG_MODULE_REGISTER(isotp, CONFIG_CAN_ISOTP_LOG_LEVEL);`